boards/arm/ks_zion/doc/index.rst

```diff
@@ -0,0 +1,127 @@
+.. _ks_zion_board:
+
+
+Kernelspace Zion
+################
+
+.. contents::
+
+
+Overview
+========
+
+The Kernelspace Zion is a small generic development board, featuring an
+ARM Cortex-M4 STM32F303K8 MCU with the minal required hardware layout
+for beginners to get started quickly. It's a simple open-hardware board
+that has been thought to be assembled easily at home.
+
+Specifications, schematic and pcb layout are available at the related
+`Zion page <https://kernel-space.org/hardware/zion.html>`_
```

Comment:
Open question:
This seem to be a very niche/diy board. @kartben Do we have a policy on supporting such boards ?
How do we ensure board support will be maintained given the community of users will be quite thin.